### ISSUE TITLE
Uncaught TypeError: Cannot convert undefined or null to object

### ISSUE THREAD

**Issue by nie-xin** [danielbrodin/atom-project-manager#266]
[Enter steps to reproduce below:]
1. ...
2. ...

**Atom Version**: 1.10.2
**Electron Version**: 0.37.8
**System**: Mac OS X 10.11.6
**Thrown From**: [project-manager](https://github.com/danielbrodin/atom-project-manager) package, v3.2.1
### Stack Trace

Uncaught TypeError: Cannot convert undefined or null to object

```
At /Users/xin/.atom/packages/project-manager/lib/stores/FileStore.js:49

TypeError: Cannot convert undefined or null to object
    at /Users/xin/.atom/packages/project-manager/lib/stores/FileStore.js:49:26
    at /Users/xin/.atom/packages/project-manager/node_modules/season/lib/cson.js:180:53
    at tryToString (fs.js:414:3)
    at FSReqWrap.readFileAfterClose [as oncomplete] (fs.js:401:12)
```
### Commands

```
  7x -1:04.5.0 core:backspace (atom-text-editor.editor.mini.is-focused)
     -0:54.3.0 core:confirm (atom-text-editor.editor.mini.is-focused)
```
### Config

``` json
{
  "core": {
    "customFileTypes": {
      "source.ini": [
        ".buckconfig",
        ".flowconfig"
      ],
      "source.json": [
        ".arcconfig",
        "BUCK.autodeps"
      ],
      "source.python": [
        "BUCK"
      ]
    },
    "disabledPackages": [
      "highlight-column",
      "linter-sass-lint",
      "tree-view"
    ],
    "themes": [
      "nucleus-dark-ui",
      "robin-hood-syntax"
    ]
  }
}
```
### Installed Packages

``` coffee
# User
advanced-open-file, v0.16.4 (inactive)
atom-ternjs, v0.15.0 (active)
autoclose-html, v0.23.0 (active)
csslint, v1.1.5 (active)
editorconfig, v1.4.1 (active)
file-icons, v1.7.19 (active)
gaia-dark-syntax, v1.0.0 (inactive)
genesis-ui, v0.5.0 (inactive)
git-control, v0.8.2 (inactive)
git-plus, v5.17.0 (inactive)
git-time-machine, v1.5.3 (inactive)
highlight-column, v0.5.1 (inactive)
highlight-line, v0.11.1 (active)
highlight-selected, v0.11.2 (active)
language-babel, v2.38.0 (active)
linter, v1.11.16 (active)
linter-eslint, v7.3.1 (active)
linter-sass-lint, v1.7.1 (inactive)
merge-conflicts, v1.4.4 (active)
minimap, v4.25.0 (active)
nucleus-dark-ui, v0.9.5 (active)
pigments, v0.34.0 (active)
project-manager, v3.2.1 (active)
robin-hood-syntax, v1.0.0 (active)
sort-lines, v0.14.0 (inactive)
synced-sidebar, v0.4.3 (active)
terminal-plus, v0.14.5 (active)
tool-bar, v1.0.1 (active)
tool-bar-almighty, v0.7.0 (active)
tree-view-git-modified, v0.7.1 (active)
tree-view-git-status, v1.1.0 (active)
atom-dark-syntax, v0.27.0 (inactive)
atom-dark-ui, v0.52.0 (inactive)
atom-light-syntax, v0.28.0 (inactive)
atom-light-ui, v0.44.0 (inactive)
base16-tomorrow-dark-theme, v1.1.0 (inactive)
base16-tomorrow-light-theme, v1.1.1 (inactive)
one-dark-ui, v1.5.0 (inactive)
one-light-ui, v1.5.0 (inactive)
one-dark-syntax, v1.3.0 (inactive)
one-light-syntax, v1.3.0 (inactive)
solarized-dark-syntax, v1.0.2 (inactive)
solarized-light-syntax, v1.0.2 (inactive)
about, v1.5.3 (active)
archive-view, v0.61.1 (active)
autocomplete-atom-api, v0.10.0 (active)
autocomplete-css, v0.11.2 (active)
autocomplete-html, v0.7.2 (active)
autocomplete-plus, v2.31.1 (active)
autocomplete-snippets, v1.11.0 (active)
autoflow, v0.27.0 (inactive)
autosave, v0.23.1 (active)
background-tips, v0.26.1 (active)
bookmarks, v0.41.0 (active)
bracket-matcher, v0.82.1 (active)
command-palette, v0.38.0 (inactive)
deprecation-cop, v0.54.1 (active)
dev-live-reload, v0.47.0 (active)
encoding-selector, v0.22.0 (active)
exception-reporting, v0.39.0 (active)
find-and-replace, v0.201.0 (inactive)
fuzzy-finder, v1.3.0 (active)
git-diff, v1.1.0 (active)
go-to-line, v0.31.0 (inactive)
grammar-selector, v0.48.1 (active)
image-view, v0.58.2 (active)
incompatible-packages, v0.26.1 (active)
keybinding-resolver, v0.35.0 (active)
line-ending-selector, v0.5.0 (active)
link, v0.31.1 (inactive)
markdown-preview, v0.158.0 (active)
metrics, v0.53.1 (active)
notifications, v0.65.0 (active)
open-on-github, v1.2.0 (inactive)
package-generator, v1.0.0 (inactive)
settings-view, v0.242.2 (active)
snippets, v1.0.2 (active)
spell-check, v0.67.1 (active)
status-bar, v1.4.1 (active)
styleguide, v0.47.0 (active)
symbols-view, v0.113.0 (inactive)
tabs, v0.100.2 (active)
timecop, v0.33.2 (active)
tree-view, v0.208.2 (active)
update-package-dependencies, v0.10.0 (active)
welcome, v0.34.0 (active)
whitespace, v0.33.0 (active)
wrap-guide, v0.38.1 (active)
language-c, v0.52.1 (active)
language-clojure, v0.21.0 (active)
language-coffee-script, v0.47.2 (active)
language-csharp, v0.12.1 (active)
language-css, v0.37.1 (active)
language-gfm, v0.88.0 (active)
language-git, v0.15.0 (active)
language-go, v0.42.1 (active)
language-html, v0.45.1 (active)
language-hyperlink, v0.16.0 (active)
language-java, v0.23.0 (active)
language-javascript, v0.119.0 (active)
language-json, v0.18.2 (active)
language-less, v0.29.5 (active)
language-make, v0.22.2 (active)
language-mustache, v0.13.0 (active)
language-objective-c, v0.15.1 (active)
language-perl, v0.35.0 (active)
language-php, v0.37.2 (active)
language-property-list, v0.8.0 (active)
language-python, v0.45.0 (active)
language-ruby, v0.69.0 (active)
language-ruby-on-rails, v0.25.0 (active)
language-sass, v0.56.0 (active)
language-shellscript, v0.22.4 (active)
language-source, v0.9.0 (active)
language-sql, v0.23.0 (active)
language-text, v0.7.1 (active)
language-todo, v0.28.0 (active)
language-toml, v0.18.0 (active)
language-xml, v0.34.9 (active)
language-yaml, v0.26.0 (active)

# Dev
No dev packages
```

**Comment by danielbrodin** [comment]
Did this happen the first time you used it or have you done something to the `projects.cson` file or something?

**Comment by curtisblackwell** [comment]
This happened to me as soon as I installed it. Didn't do anything else.

**Comment by telth** [comment]
Looks like this error only pops up if your projects.cson is empty. Soon as you've saved a single project it stops.

**Comment by curtisblackwell** [comment]
That sounds likely. I haven't had an error since adding a project. Might be good to catch it so as not to scare off first-time users.

**Comment by danielbrodin** [comment]
Thought I had managed that, but guess I have to add an extra check. Thanks for letting me know

**Comment by 20after4** [comment]
For me this happened when setting up my second php project.

**Comment by Gert-dev** [comment]
Just wanted to mention this can also happen with non-empty `projects.cson` files.

I have no idea how I just ran into it, but I had a project open in a background window. It loaded fine and I was doing some tests with it, but when I came back after leaving it open for a while this error was shining in the upper right corner.

Not sure if it matters, but the project has nothing special about it, except that it has some custom settings attached under a `php` key (right underneath `devMode` and `template`) and it has the relative path `~/Test`, which I manually changed from absolute to relative for testing purposes.
